index.js

```diff
@@ -57,6 +58,10 @@ function setup(options) {
     return Promise.reject('Must specify bucket');
   }
 
+  if (!config.acl) {
+    config.acl = 'private';
```

Comment:
Since the previous behavior was to use "public-read", let's set the default to "public-read".